src/components/wordbank/Wordbank.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @jsx React.DOM
+ */
+
+/*
+ == BSD2 LICENSE ==
+ Copyright (c) 2015, Tidepool Project
+
+ This program is free software; you can redistribute it and/or modify it under
+ the terms of the associated License, which is identical to the BSD 2-Clause
+ License as published by the Open Source Initiative at opensource.org.
+
+ This program is distributed in the hope that it will be useful, but WITHOUT
+ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ FOR A PARTICULAR PURPOSE. See the License for more details.
+
+ You should have received a copy of the License along with this program; if
+ not, you can obtain one from Tidepool Project at tidepool.org.
+ == BSD2 LICENSE ==
+ */
+'use strict';
+/* jshint unused: false */
+
+var React = require('react');
+var _ = require('lodash');
+var dataHelper = require('../../core/userDataHelper');
```

Comment:
I could export a function that accepts a wordbank array as an argument, and returns the var that is currently exported. How does that sound?